docs/DeveloperGuide.md

```diff
@@ -294,139 +294,110 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 (For all use cases below, the **System** is the `CohortConnect` and the **Actor** is the `user`, unless specified otherwise)
 
-2. **Use case: Find a contact using name**
-
-    **MSS**
-    
-    1.  User enters command to find a contact by name
-    2.  CohortConnect shows list of contacts with matching name
-
-        Use case ends.
+**Use Case 4: Find a contact using name**
+ **MSS**
```

Comment:
I think there should be an extra empty line before the MSS! It currently looks like this, same goes for the other use cases.

![Screenshot 2021-09-27 at 4 32 12 PM](https://user-images.githubusercontent.com/31569631/134873078-16c5e530-4792-4edb-8601-2589a9c40d67.png)

Reply:
Oh another thing, I was thinking maybe you could un-bold the extensions as well, I think it would help with readability!

![image](https://user-images.githubusercontent.com/31569631/134874365-c65ecec5-ab3b-41c2-8b90-ef247714ab6a.png)